core/src/main/java/com/google/errorprone/bugpatterns/UnnecessarilyFullyQualified.java

```diff
@@ -160,6 +179,23 @@ && getSymbol(tree.getExpression()) instanceof ClassSymbol) {
         treePaths.add(path);
       }
 
+      private boolean isBadNestedClass(MemberSelectTree tree) {
+        return BadImport.BAD_NESTED_CLASSES.contains(tree.getIdentifier().toString());
+      }
+
+      private boolean isExemptedEnclosingType(MemberSelectTree tree) {
```

Comment:
Does it make sense to early-exit if the flag is not set? Or, is the cost negligible? :thinking: 

```suggestion
      private boolean isExemptedEnclosingType(MemberSelectTree tree) {
        if (exemptedEnclosingTypes.isEmpty()) {
          return false;
        }
```

Reply:
Either way is fine with me, I think the cost is probably negligible.

Reply:
Leaving it as-is then :+1: